app/models/student_work_group.rb

```diff
@@ -27,10 +27,19 @@ class StudentWorkGroup < ApplicationRecord
   belongs_to :work_group
 
   validate :user_role
+  validates :work_group, presence: true
+  validate :only_one_student_turn
 
   private
 
   def user_role
     errors.add(:not_student, 'Teachers cannot create Student Work Groups') if user&.teacher?
   end
+
+  def only_one_student_turn
+    return if work_group.nil?
```

Comment:
same as above

Reply:
without this the test above fails, is there another way? 
<img width="1017" alt="Screen Shot 2020-12-29 at 10 36 33" src="https://user-images.githubusercontent.com/56534210/103252894-07e71580-49c2-11eb-85ac-ad8d856d514f.png">

Reply:
weird, though I guess we had a similar issue in others
then I guess you can keep this condition, but then you have to add another space (guard clause, rubocop)

Reply:
sure! Changing that